plugins/USBPrinting/USBPrinterOutputDevice.py

```diff
@@ -563,19 +563,19 @@ def _sendNextGcodeLine(self):
             line = line[:line.find(";")]
         line = line.strip()
 
-        # Don't send empty lines. But we do have to send something, so send m105 instead.
-        if line == "":
+        # Don't send empty lines. But we do have to send something, so send
+        # m105 instead.
+        # Don't send the M0 or M1 to the machine, as M0 and M1 are handled as
+        # an LCD menu pause.
+        if line == "" or line == "M1" or line == "M1":
```

Comment:
There is a little mistake here; It should be line == "M0" or line == "M1".

I will fix that in a sperate commit.